src/js/__tests__/__snapshots__/tslint-test.ts.snap

```diff
@@ -523,6 +523,7 @@ exports[`tslint snapshot should be up to date 1`] = `
 /plugins/services/src/js/service-configuration/PodNetworkConfigSection.tsx - Shadowed name: 'endpoints'
 /plugins/services/src/js/service-configuration/PodNetworkConfigSection.tsx - Shadowed name: 'memo'
 /plugins/services/src/js/service-configuration/PodStorageConfigSection.tsx - Use a conditional expression instead of assigning to 'type' in multiple places.
+/plugins/services/src/js/service-configuration/ServiceGeneralConfigSection.tsx - == should be ===
```

Comment:
let's please rather reduce the overall number of lints in every PR.

Reply:
✅